TUW_Radar_Image_Compositing/JSON/jrc_uc1_temp.json

```diff
@@ -0,0 +1,207 @@
+      "arguments": {
+        "data": {
+          "from_node": "filtertemporal2"
+        },
+        "dimension": "t",
+        "reducer": {
+          "process_graph": {
+            "mean2": {
+              "process_id": "mean",
+              "arguments": {
+                "data": {
+                  "from_parameter": "data"
+                }
+              },
+              "result": true
+            }
+          }
+        }
+      },
+      "description": "Calculate the mean over April 2017 resulting in a single image of mean values"
+    },
+    "reducedimension3": {
+      "process_id": "reduce_dimension",
+      "arguments": {
+        "data": {
+          "from_node": "filtertemporal3"
+        },
+        "dimension": "t",
+        "reducer": {
+          "process_graph": {
+            "mean3": {
+              "process_id": "mean",
+              "arguments": {
+                "data": {
+                  "from_parameter": "data"
+                }
+              },
+              "result": true
+            }
+          }
+        }
+      },
+      "description": "Calculate the mean over May 2017 resulting in a single image of mean values"
+    },
+    "renamelabels1": {
+      "process_id": "rename_labels",
+      "arguments": {
+        "data": {
+          "from_node": "reducedimension1"
+        },
+        "dimension": "bands",
+        "target": [
+          "R"
+        ]
+      },
+      "description": "Rename the VV band label to R"
+    },
+    "renamelabels2": {
+      "process_id": "rename_labels",
+      "arguments": {
+        "data": {
+          "from_node": "reducedimension2"
+        },
+        "dimension": "bands",
+        "target": [
+          "G"
+        ]
+      },
+      "description": "Rename the VV band label to G"
+    },
+    "renamelabels3": {
+      "process_id": "rename_labels",
+      "arguments": {
+        "data": {
+          "from_node": "reducedimension3"
+        },
+        "dimension": "bands",
+        "target": [
+          "B"
+        ]
+      },
+      "description": "Rename VV band label to B"
+    },
+    "mergecubes1": {
+      "process_id": "merge_cubes",
+      "arguments": {
+        "cube1": {
+          "from_node": "renamelabels1"
+        },
+        "cube2": {
+          "from_node": "renamelabels2"
+        }
+      },
+      "description": "Merge R and G data cubes to one with both images in the band labels R and G"
+    },
+    "mergecubes2": {
+      "process_id": "merge_cubes",
+      "arguments": {
+        "cube1": {
+          "from_node": "mergecubes1"
+        },
+        "cube2": {
+          "from_node": "renamelabels3"
+        }
+      },
+      "description": "Merge RG and B data cubes to one with all three images in the band labels R, G and B"
+    },
+    "saveresult1": {
+      "process_id": "save_result",
+      "arguments": {
+        "data": {
+          "from_node": "mergecubes2"
+        },
+        "format": "GTIFF-THUMB",
```

Comment:
Are you sure this is really GTIFF-THUMB? That is a GEE only thing usually...

Reply:
That is an error indeed. Should be GTIFF instead